### ISSUE TITLE
Bug : swipe up or down on modals

### ISSUE THREAD

**Issue by bitzi** [adospace/rive-app#4]
Hi Ado,

MauiReactor is just amazing, thank you for your great work. I was testing this Rive app and noticed a bug: when you tap on the user profile icon then after the pop up displays,  if you swipe up or down you can see that it scrolls the list view underneath the pop up. 
I guess this should not happen?

**Comment by adospace** [comment]
Yes, surely it shouldn't happen, of course, this is just a showcase app not intended to be fully working but I'll take a look.
Thank you for your interest!
